test/Microsoft.AspNet.Mvc.IntegrationTests/ModelBindingTestHelper.cs

```diff
@@ -67,6 +69,21 @@ public static IObjectModelValidator GetObjectValidator(MvcOptions options = null
                     metadataProvider);
         }
 
+        public static void UpdateOptionsToEnsureNothingFollows<TModelBinder>(MvcOptions options)
+        {
+            var index = 0;
+            for (; index < options.ModelBinders.Count; index++)
+            {
+                if (typeof(TModelBinder) == options.ModelBinders[index].GetType())
+                {
+                    options.ModelBinders.Insert(index + 1, new AssertIfCalledModelBinder());
+                    return;
+                }
+            }
+
+            Assert.False(true, $"Unable to find { typeof(TModelBinder).FullName } in { nameof(MvcOptions) }.");
+        }
```

Comment:
This meaning the assertion? It's a guard to ensure the setup does what the test expects.